subsys/bluetooth/mesh/access.c

```diff
@@ -301,22 +306,25 @@ static void mod_init(struct bt_mesh_model *mod, struct bt_mesh_elem *elem,
 	}
 
 	if (mod->cb && mod->cb->init) {
-		mod->cb->init(mod);
+		*err = mod->cb->init(mod);
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/blob/d2c74eb98754e824196541cc6fa6a420fb4ed611/subsys/bluetooth/mesh/access.c#L340-L348

should add check `*err is NULL` ???

Reply:
Either that or pass an err to the unprovision case

Reply:
Should this call be removed, now that we have the reset callback in the config server?

Reply:
Yes, I think so